README.md

```diff
@@ -1 +1,74 @@
-# tracker
\ No newline at end of file
+# CAMERA-OBJECT-TRACKER-SERVICE
+
+This is a [Viam module](https://docs.viam.com/extend/modular-resources/) providing a model of vision service for tracking object using feature-based matching.
+
+
+## Getting started
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/modular-resources/#configuration) and select the `viam:vision:re-id-object-tracker` model from the [`camera-object-tracking-service` module](https://app.viam.com/module/viam/camera-object-tracking-service).
+This module implements the following methods of the [vision service API](https://docs.viam.com/services/vision/#api):
+
+- `GetDetections()`: returns the bounding boxes with the unique id as label and the matching confidence.
+- `CaptureAllFromCamera()`: returns the image and detections all together, given a camera name.
+
+The tracking problem consists of two parts: detection and visual matching. The `camera-object-tracking-service` module provides the necessary logic to perform object tracking, given a detector and a visual matcher (i.e., an embedder). The tracker can use either a default, built-in general embedder or an ML model service.
```

Comment:
Change the last sentence to: If you don't configure an embedder (ML model service) or detector (vision service), defaults will be used.

Reply:
You could even add "Defaults are chosen to be general purpose."